Created 2017-11-28-data-science-assessment-challenges.md

kariljordan opened the pull request:
@mkcor final blog post!
@tracykteal please review for merge

mkcor commented on _posts/2017-11-28-data-science-assessment-challenges.md:
```diff
@@ -0,0 +1,57 @@
+__Challenges Assessing Data Science Education__ 
+
+Working groups identified challenges for assessing data science education at the object level (i.e., what should this assessment
+consist of?) and at the meta level (i.e., what favors or hinders the application of assessment?)
```
Period missing at end of sentence.
And, maybe, there should be no line break (between 42 and 43).

kariljordan replied:
Fixed

mkcor commented on _posts/2017-11-28-data-science-assessment-challenges.md:
```diff
@@ -0,0 +1,57 @@
+
+In the second part of the meeting, we split into breakout sessions in pairs to discuss the challenges of assessing data science education with respect to the Carpentries workshops. Brainstorming in parallel lets us cover more ground (breadth), while interacting one-on-one lets us explore different avenues (depth).
+
+One pair focused on the industry perspective, another on the education system, and the third on assessment practices. Kari offered a list of questions to frame the discussion. The questions were pulled from the meeting she attended.
```
Wondering if we could drop "The questions were pulled from the meeting she attended." since we repeat it line 45 "(pulled from South Big Data Hub's Data Divide workshop)".

kariljordan replied:
Agreed!

tracykteal commented on _posts/2017-11-28-data-science-assessment-challenges.md:
```diff
@@ -0,0 +1,57 @@
+show_meta: true
+authors: ["Marianne Corvellec " "Kari L. Jordan "]
+---
+On November 15, the [virtual assessment network](https://groups.google.com/a/carpentries.org/forum/#!forum/assessment-network) held its fifth meeting. It was organized and hosted online by Kari Jordan, and six community members attended. The conversation revolved around data science education. 
```
The link to the Google groups goes to a page where you have to be a member to see anything. Is there any other page that describes the virtual assessment network?

kariljordan replied:
Right now there's a repo and then our [Assessment page](http://www.datacarpentry.org/assessment/). I could add information about the assessment network to the assessment page, or I could add a new page under the Get Involved tab of the website. I like the second option (page under get involved tab)

tracykteal commented:
Thanks for putting this together. There's great information here. Talked with @kariljordan about adding a couple headers to highlight the two different sections.

kariljordan commented:
@tracykteal I added the headings and removed the SQL paragraph. Added an intro sentence. Once [this PR](https://github.com/datacarpentry/datacarpentry.github.io/pull/341) is reviewed and merged I'll link the assessment network page to this blog post, and then we can merge. Then I'll add it to SWC's blog. Thanks! (cc @mkcor)
